interface_specifications/pe_openapi/apis/v1/e0_pe_presentation_statuses_post.md

```diff
@@ -0,0 +1,45 @@
+### Presentation Status POST
+
+`POST /pe/v1/presentations/{presentation_id}/statuses`
+
+#### Description
+
+Verifier can update the status from `SUBMITTED` to `ACCEPTED` by calling this API.
+
+
+#### Parameters
+
+Path Variables:
+* `presentation_id`: "a30e3b91-fb77-4d22-95fa-871689c322e2"
+
+  Request Body:
+  ```json
+  {
+    "thread":{
+      "id": "f1ca8245-ab2d-4d9c-8d7d-94bf310314ef"
+    },
+    "presentation_id": "a30e3b91-fb77-4d22-95fa-871689c322e2", 
```

Comment:
presentation_id is already in the path, so no need to have it in the request body

Reply:
We need this presentation id to be present in the object to validate it.
We can indeed copy the id present in the path to the object on the backend side, in controller or filters.

So, shall we remove the redundancy ?

Reply:
yes please, let's not border the caller with having to supply the same info in 2 locations during a request

Reply:
removing from request body.